pkgs/by-name/cc/cc1541/package.nix

```
@@ -7,13 +7,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "cc1541";
-  version = "4.1";
+  version = "unstable-2024-02-29";
 
   src = fetchFromBitbucket {
     owner = "ptv_claus";
     repo = "cc1541";
-    rev = finalAttrs.version;
-    hash = "sha256-b8cEGC3WxrjebQjed/VD9SIWkiQpNaE2yW+bQRCtmSs=";
+    rev = "f51ba77";
```

```suggestion
    rev = "f51ba77478b0e135b6e6ace24180a708850295bb";
```
You must use the full commit hash.